app/helpers/batch_edits_helper.rb

```diff
@@ -6,6 +6,6 @@ def batch_delete
   end
 
   def render_check_all
-    render partial: 'batch_edits/check_all' unless params[:controller].match("my/collections")
+    render partial: 'batch_edits/check_all' unless params[:controller] =~ /my\/collections/
```

Comment:
@hectorcorrea ok. That issue makes sense, but the regex should still work. Although, it should probably be `/^my\/collections$/` if we want to be exact. I don't have any opinions one way or the other... we can add an exception to Rubocop, or update the regex.